### PR TITLE
Migrate serato/serato-sendgrid to Github Actions

### PR DESCRIPTION
**Pull request by aosmakserato**
- Added the workflow file
- Removed `travis.yml`
- Upgraded `phpstan` and suppressed several errors that the newer version was throwing. We will have to refactor the code next time we touch Mailer.php.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	},
 	"require-dev": {
 		"phpunit/phpunit": "~7.0",
-		"phpstan/phpstan-phpunit": "^0.11.0",
+		"phpstan/phpstan-phpunit": "^0.12.22",
```
Comment (aosmakserato) [inline composer.json:12]: Unfortunately we can't upgrade to 1.x because it does not support PHP 7.1

**Review by sameeraroshan-serato — APPROVED** [review]
Nice one @aosmakserato

File: phpstan.neon [hunk 2]
```diff
@@ -3,5 +3,10 @@ parameters:
...
-    includes:
-        - vendor/phpstan/phpstan-phpunit/extension.neon
\ No newline at end of file
+# We started getting those errors after upgrading phpstan-phpunit to 0.12. Consider fixing those next time we touch Mailer.php
+    ignoreErrors:
+        -
+            message: '#.*no value type specified in iterable type array\.$#'
+            path: src/Mailer.php
+includes:
+    - vendor/phpstan/phpstan-phpunit/extension.neon
```
Comment (AnnieCSerato) [inline phpstan.neon:12]: indentation?

Comment (aosmakserato) [reply phpstan.neon:12]: Nope, I had to change it because the new version changed the format.

**Review by AnnieCSerato — APPROVED** [review]
Nice 👍

**Review by mshahidi — APPROVED** [review]

**Pull request merged**